staging/src/kubevirt.io/client-go/kubecli/vmi.go

```diff
@@ -147,6 +148,10 @@ func RequestFromConfig(config *rest.Config, resource, name, namespace, subresour
 		return nil, fmt.Errorf("Unsupported Protocol %s", u.Scheme)
 	}
 
+	u.Path = path.Join(
+		u.Path,
+		fmt.Sprintf("/apis/subresources.kubevirt.io/%s/namespaces/%s/%s/%s/%s", v1.ApiStorageVersion, namespace, resource, name, subresource),
+	)
 	u.Path = fmt.Sprintf("/apis/subresources.kubevirt.io/%s/namespaces/%s/%s/%s/%s", v1.ApiStorageVersion, namespace, resource, name, subresource)
```

Comment:
This line would need to be removed otherwise it will overwite `u.Path`

Reply:
Thank you very much, it slipped back in there when I was typing up the PR and wanted to get the variable value. I have fixed it.